FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -770,77 +1116,151 @@ components:
+        timeOfReplacement:
+          description: Defaults to TIME OF REGISTRATION unless there has been a reset from an administrator to start tracking a different period of time. Tracks the last time that a component was repaired, replaced, or any other maintenance has been performed, thus requiring a reset in the lifetime statistics below / MTBF time.  
+          type: string
+          format: date-time
+          
+        sessionRunningTimeSeconds:
+          description: If ONLINE, this is the amount of time (ms) passed between when the component started being ONLINE to the present. Otherwise 0. Function of (NOW - Component Startup)
+          type: integer
+          format: int64
+        
+        totalRunningTimeSeconds:
+          description: This is the amount of time that the component has been ONLINE since it has been registered with FRAIHMWORK. Could be seen as the sum of all SESSION RUNNING TIMEs since the TIME OF REGISTRATION. Used for tracking time used to measure MTBF 
+          type: integer
+          format: int64
+          
+        registrationLifetimeSeconds:
+          description: Total time passed since TIME OF REGISTRATION. Total amount of time that the FRAIHMWORK CORE has been aware of this component. Function is (NOW - TIME OF REGISTRATION)
+          type: integer
+          format: int64
+
+        totalUptime:
+          description: A percentage which is a function of TOTAL RUNNING TIME / REGISTRATION LIFETIME
+          type: number
+          format: double
+          
+        totalPerformanceRunningTimeSeconds:
+          description: This is the amount of time that the component has been ONLINE since the TIME OF PERFORMANCE. Could be seen as a sum of all SESSION LIFETIMEs since the TIME OF PERFORMANCE. Used for tracking data over a specific period of time.
+          type: integer
+          format: int64
+
+        performanceLifetimeSeconds:
+          description: Total time passed since TIME OF PERFORMANCE . Total amount of time that the FRAIHMWORK CORE has been aware of this component since a manual reset. Function is (NOW - TIME OF PERFORMANCE)
+          type: integer
+          format: int64
+
+        performanceUptime:
+          description: A percentage which is a function of (TOTAL PERFORMANCE RUNNING TIME) / (PERFORMANCE LIFETIME).
+          type: number
+          format: double
+          
+        totalReplacementRunningTimeSeconds:
+          description: This is the amount of time that the component has been ONLINE since the TIME OF REPLACEMENT. Could be seen as a sum of all SESSION LIFETIMEs since the TIME OF REPLACEMENT. Used for tracking data over a specific period of time.
+          type: integer
+          format: int64
+
+        replacementLifetimeSeconds:
+          description: Total time passed since TIME OF REPLACEMENT. Total amount of time that the FRAIHMWORK system has been aware of this component since a replacement. Function is (NOW - TIME OF REPLACEMENT)
```

Comment:
Pedantic, but the only place that specifies units is the field name.

Reply:
Addressed here and in a few other places